app/src/pages/FormPage.js

```diff
@@ -102,14 +136,17 @@ const FormPage = ({ navigation, route }) => {
   const handleOnSubmitForm = async (values, refreshForm) => {
     try {
       const submitData = {
-        form: selectedForm.id,
+        form: currentFormId,
```

Comment:
formId @wayangalihpratama 
https://github.com/akvo/national-wash-mis/blob/f80cfb8b65da219e6471ccf119e1ccab60e6f915/backend/api/v1/v1_mobile/views.py#L69-L76

Reply:
This is for sqlite db, and we use form as form id column name.

Reply:
We don't have geo on our sqlite. 
I need to modify the data point table.
What is the format for geo? Is it something like `lat|lng`?

Reply:
array `["lat","long"]` @wayangalihpratama

Reply:
btw, you need also to check if the geo is meta or not. If there's no geo, remove geo key. (so; add geo key after submitData is defined)

Reply:
Hi mas @dedenbangkit, I have pushed some updates.
We will need to down -v first, because I add a new column to SQLite DB.

btw, for the SQLite db, I set geo as null if not meta.
And for sync API, that will be received as `null`?
Sync parameter: https://github.com/akvo/nmis-mobile/blob/d42980da57b7d87e975092570e47141ad1c21476/app/src/lib/background-task.js#L91-L106